Tests/test_image_getdominantcolors.py

```diff
@@ -0,0 +1,18 @@
+from .helper import hopper
+
+
+def test_getdominantcolors():
+    def getdominantcolors(mode):
+        im = hopper(mode)
```

Comment:
I see what you mean about it being slow, `pytest Tests/test_image_getdominantcolors.py` takes ~5.67s on my Mac.

Resizing to `(10, 10)` reduces it to 0.23s.
Resizing to `(20, 20)` reduces it to 0.41s.

Do you think 10x10 might be too small to be useful?

```suggestion
        im = hopper(mode)
        im.thumbnail((10, 10))
```

Reply:
Small sizes like that give a very rough approximation especially if you're looking for a palette that's more than 3 colors.  The colors returned tend to be much darker.  I found that `(100,100)` gives somewhat decent results.